card/lib/card/format/nest/subformat.rb

```diff
@@ -5,8 +5,9 @@ module Subformat
         def subformat subcard
           subcard = Card.fetch(subcard, new: {}) if subcard.is_a?(String)
           self.class.new subcard, root: @root, parent: self, depth: @depth + 1,
-                                  form: @form, mode: @mode,
-                                  context_names: @context_names
+                         form: @form,
+                         mode: @mode, native_mode: @native_mode,
+                         context_names: @context_names
```

Comment:
More scrooginess from me here, because I was hoping to remove most of these and make them lazy-load from the parent)